Add-IN/ACDC4Robot/core/joint.py

```diff
@@ -15,11 +17,16 @@ class Joint():
     Joint class for joint
     """
 
-    def __init__(self, joint: adsk.fusion.Joint) -> None:
+    def __init__(self, joint: Union[adsk.fusion.Joint, adsk.fusion.AsBuiltJoint]) -> None:
         self.joint = joint
         self.name = joint.name
-        self.parent = joint.occurrenceTwo # parent link of joint
-        self.child = joint.occurrenceOne
+        try:
```

Comment:
In some cases, like when creating a rigid joint to an origin (i.e. lock to world), fetching the occurrences throws an exception.